android/app/build.gradle

```diff
@@ -127,6 +127,7 @@ android {
         ndk {
             abiFilters "armeabi-v7a", "x86", "arm64-v8a", "x86_64"
         }
+        missingDimensionStrategy 'react-native-camera', 'general'
```

Comment:
Is this needed because the migration from 1.x to 2.x version? Or is it auto-generated?

Reply:
I added it because now it's necessary. Without this the project is not built because the library wants it set to "general" or "MLKit". 
I leave you the link of the guide for installing Android where it says 
https://react-native-community.github.io/react-native-camera/docs/installation